profiles/kentik_snmp/cisco/cisco-wlc.yml

```diff
@@ -539,4 +539,16 @@ metric_tags:
   # Version of the product.
   - column:
       name: agentInventoryProductVersion
-      OID: 1.3.6.1.4.1.14179.1.1.1.14.0
+      OID: 1.3.6.1.4.1.14179.1.1.1.14.0      
+      
+ - MIB: CISCO-LICENSE-MGMT-MIB
+   table:
+      name: clmgmtLicenseInfoTable
+      OID: 1.3.6.1.4.1.9.9.543.1.2.3
+    symbols:
+      # Maximum number of entities that can use a license.
+      - OID: 1.3.6.1.4.1.9.9.543.1.2.3.1.10
+        name: clmgmtLicenseMaxUsageCount
+      # Remaining number of entities that can still use a license  
+      - OID: 1.3.6.1.4.1.9.9.543.1.2.3.1.11
+        name: clmgmtLicenseUsageCountRemaining
```

Comment:
@pkanakaraj-sf - please move this block to the main config section above line 529 and fix the indentation to ensure the YAML is in the correct syntax.